Makefile

```diff
@@ -0,0 +1,153 @@
+# Makefile to help automate tasks
+PY := bin/python
+PIP := bin/pip
+PYTEST := bin/py.test
+THEBLUES := lib/python*/site-packages/theblues.egg-link
+FLAKE8 := bin/flake8
+SPHINX := bin/sphinx-apidoc
+TOX := bin/tox
+
+help:
+	@echo "dev - install theblues in develop mode"
+	@echo "lint - check style with flake8"
+	@echo "test - run tests quickly with the default Python"
+	@echo "check - run tests against 2.7 and 3.4 and check lint."
+	@echo "test-all - run tests against 2.7 and 3.4, check lint, and test docs."
+	@echo "coverage - check code coverage quickly with the default Python"
+	@echo "docs - generate Sphinx HTML documentation, including API docs"
+	@echo "clean - remove build and python artifacts"
+
+
+#######
+# TOOLS
+#######
+$(PY):
+	virtualenv .
+
+$(PYTEST): $(PY)
+	$(PIP) install -r test-requirements.txt
+
+$(TOX): $(PY)
+	$(PIP) install -r test-requirements.txt
+
+$(FLAKE8): $(PY) 
+	$(PIP) install -r test-requirements.txt
+
+$(SPHINX): $(PYTEST) 
+	$(PIP) install -r dev-requirements.txt
+
+
+#########
+# INSTALL
+#########
+.PHONY: all
+all: venv deps dev
+
+
+venv: $(PY)
+
+$(THEBLUES):
+	$(PY) setup.py develop
+
+.PHONY: dev
+dev: venv $(THEBLUES)
+
+
+######
+# DEPS
+######
+
+.phony: sys-deps
```

Comment:
So I'm fine with sysdeps; less fine with the inconsistency. I've gone ahead and changed them to testdeps and devdeps as well.